hlx_statics/scripts/lib-helix.js

```diff
@@ -719,6 +720,34 @@ export function githubActionsBlock(doc) {
   }
 };
 
+/**
+ * parse attributes from row and added to particular blcok
+ */
+
+function parseAttribute(block) {
+  const codeTags = block.querySelectorAll('code');
+  codeTags.forEach(codeTag => {
+    const dataContent = codeTag?.textContent?.trim();
+    if (dataContent.startsWith('data-')) {
+      const attributes = dataContent.split(/data-/).filter(segment => segment.trim() !== '');
+      attributes.forEach(attribute => {
+        const [key, value] = attribute.split('=').map(part => part.trim());
+        if (key && value) {
```

Comment:
We should allow empty values because JSX (used in MDX) has a shorthand where if no value is provided, it defaults to true (see https://legacy.reactjs.org/docs/jsx-in-depth.html#props-default-to-true).

We might need to do something like this: https://github.com/AdobeDocs/adp-devsite/blob/main/hlx_statics/blocks/redoclyapiblock/redoclyapiblock.js#L37-L40

For example, if you run this test, you'll see `hideTryItPanel` got thrown away:
1. http://localhost:3000/github-actions-test/test/redocly-api-block
2. inspect
3. CTRL+F for `data-block-name="redoclyapiblock"`
4. compare the attributes added to the div vs. the attributes defined in the md file: https://github.com/AdobeDocs/adp-devsite-github-actions-test/blob/main/src/pages/test/redocly-api-block.md?plain=1#L6-L22